### ISSUE TITLE
anything new??

### ISSUE THREAD

**Issue by dengyuning** [minsangkim142/R-net#7]
It seems that the result is not good. Any new progress??

**Comment by minsangkim142** [comment]
Thanks for raising this issue!
I've been quite busy with other works recently but I'd love to get some feedback from everyone as I might have easily misinterpreted the [paper](https://www.microsoft.com/en-us/research/wp-content/uploads/2017/05/r-net.pdf) and could have written a wrong architecture. I've tried different approaches and architectures (e.g. weight sharing, 100% unidirectional RNN etc) but currently the model seems to cap at:
Exact match = 40~50 
F1 = 60 with mean training loss of around 2.
![training loss](https://user-images.githubusercontent.com/24954419/30574492-5d8bb316-9d4e-11e7-816e-b44d047e3209.png)
If you've reviewed the code and think it should be different please put forward a pull request!
Thanks

**Comment by minsangkim142** [comment]
Just clarifying confusion, the training curve above is when applied dropout of 0.2
Below is when no dropout is used.
The dev performance with dropout is 0.3/0.4 for EM/F1
and without dropout is 0.4/0.5.

![image](https://user-images.githubusercontent.com/24954419/30678794-f8c53942-9ee8-11e7-8119-834eaef3da3a.png)

**Comment by minsangkim142** [comment]
I raised another issue #12 related to this. We can keep all discussions regarding performance in this issue.
